src/library/Validate.js

```diff
@@ -36,22 +57,16 @@ function validateBlockNumOrHash(numOrHash) {
   validateBlockTxHash(numOrHash);
 }
 
-function validateBlockTxHash(hash) {
-  hash = String(hash);
-
-  if (hash === undefined || hash === '' || hash.length < 2) {
-    throw new Error('must provide a hash');
-  }
-
-  if (hash.substring(0, 2).toLowerCase() !== '0x') {
-    throw new Error("hash must start with '0x'");
-  }
-
-  if (hash.length !== 66) {
-    throw new Error(`invalid hash length of ${hash.length}, should be 64 hex characters long excluding the '0x' prefix`);
+function validateDeploymentId(deploymentId) {
+  if (!deploymentId || !RegExp('^[a-z0-9]+$', 'i').test(deploymentId)) {
```

Comment:
We have deployment IDs that include other characters (dashes, etc.), and I think we should also allow full URLs as well (by default, `multibaas.com` is appended).

Reply:
I just decided to use deployment's host (it is ok to copy full URL path) only for ID because to:

- avoid incorrect copy
- make it simple